server/application/terminal.go

```diff
@@ -60,71 +58,84 @@ func (s *terminalHandler) getApplicationClusterRawConfig(ctx context.Context, a
 	return clst.RawRestConfig(), nil
 }
 
-// isValidPodName checks that a podName is valid
-func isValidPodName(name string) bool {
+// getValidPodName checks that a podName is valid
+func getValidPodName(name string) string {
```

Comment:
I wonder what motivated you to change this signature? I honestly prefer the previous pattern with the boolean return as it expresses what the function is actually doing (validating). I think `get*` is misleading in this case as the function isn't actually retrieving anything. It is just acting on the provided value.

Reply:
Agreed. I was trying to get the CodeQL check to pass, but it didn't have any effect. Will revert to boolean.